CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## [4.0.1] - XXXX-XX-XX
+### Added
+- Added `allowEmptyTry` option to `no-empty-blocks` rule to allow empty try blocks 
```

Comment:
make it one item, showing the link to the PR as it is the case for other entries in the changelog, and the three options as sub-items